### ISSUE TITLE
Outreacht 2024: add stuff on @Siddhi-agg 's work  

### ISSUE THREAD

**Issue by cuihtlauac** [ocaml/ocaml.org#2736]
@shonfeder @NathanReb : File `data/outreachy.yml` is still missing text on what was done by @Siddhi-agg outreachy internship in summer 2024

**Comment by NathanReb** [comment]
Yes, sorry for the long delay and thank you for the reminder. I'll add that in the next few days!
